contracts/access/TimelockController.sol

```diff
@@ -93,33 +93,42 @@ contract TimelockController is AccessControl {
      */
     receive() external payable {}
 
+    /**
+     * @dev Returns whether an id correspond to a registered operation. This
+     * includes both Pending, Ready and Done operations.
+     */
+    function isOperation(bytes32 id) public view virtual returns (bool pending) {
```

Comment:
I thought that if someone was to overload the isOperationPending / isOperationReady / isOperationDone in a way that would impact the contract's orchastration, then it would be good to have this virtual function, as the corresponding logic would likely also have overriden for changes to the other functions to ben consistent